explorer/cnquery_explorer.proto

```diff
@@ -28,15 +28,24 @@ message QueryGroup {
 }
 
 message QueryPack {
-  // FIXME: DEPRECATED, remove in v9.0 vv
+  // FIXME: DEPRECATED, remove in v9.0 vv and reserve number
   // This is replaced by the field "filters"
   map<string,Mquery> asset_filters = 7; // key == checksum
   // ^^
 
+  // FIXME: DEPRECATED, remove in v9.0 vv and reserve number
+  // This is replaced by the field "filters"
+  repeated string old_filters = 43;
```

Comment:
I think this should be called `deprecated_filters`

Reply:
much better, thank you